### PR TITLE
Added components for the first page and questions to increase iteration in vue

### PR DESCRIPTION
**Pull request by lewisdylan**

### CODE REVIEW

File: app/controllers/api/questions_controller.rb [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 module Api
   class QuestionsController < BaseController
+    def index
+      question = Question.where(entry: true).first!
```
Comment (bumi) [inline app/controllers/api/questions_controller.rb]: what is this method for? 
you should get the first question from the iteration object `starting_question_id`

Comment (bumi) [reply app/controllers/api/questions_controller.rb]: also the index method (GET /iterations) indicates that it returns a list of iterations, so I would not return only one question here.

Comment (lewisdylan) [reply app/controllers/api/questions_controller.rb]: for the index method, i was thinking that whenever a user decides to start a new iteration they should be redirected to this method where they get just the first question

Comment (bumi) [reply app/controllers/api/questions_controller.rb]: hmm. I see. 
the iteration actually returns the `starting_question_id` 
we should get this when creating an iteration or when loading an iteration (GET iterations/[id])

do we then still need this extra endpoint?

Comment (musayann) [reply app/controllers/api/questions_controller.rb]: Yeah I took, think this is absolute cause it was implemented before @bumi you implemented the `starting_question_id` method

Comment (bumi) [reply app/controllers/api/questions_controller.rb]: ok. I see, but then let's clean it up.

**Comment by bumi** [comment]
fix travis and we merge this one to continue on this one

**Pull request merged**